autocertkit/networkconf.example

```diff
@@ -12,6 +12,11 @@ eth3 = 1,[200]
 
 # static_<network _id>_<vlan_id> = <ip_start>,<ip_end>,<netmask>,<gw>
 
+# In case, management network is not having DHCP and management interface
+# is not specified above ( management interface is not being certified),
+# It is mandatory to specify static addressing, i.e.:
+#  static_management = 192.168.0.2,192.168.0.10,255.255.255.0,192.168.0.1
```

Comment:
How is this different from static_0_0 ?
Is it okay to accept IP range for static 'management' IP? if it is management, it should be fixed and set up already on host. Hence it should be 'fixed' single IP.